src/Tribe/Attendees_Table.php

```diff
@@ -194,6 +172,25 @@ public function column_order_status( $item ) {
 	}
 
 	/**
+	 * Retrieves the order id for the specified table row item.
+	 *
+	 * In some cases, such as when the current item belongs to the RSVP provider, an
+	 * empty string may be returned as there is no order screen that can be linekd to.
+	 *
+	 * @param array $item
+	 *
+	 * @return string
+	 */
+	public function get_order_id_url( array $item ) {
+		// Backwards compatibility
+		if ( empty( $item['order_id_url'] ) ) {
+			$item['order_id_url'] = esc_url( get_edit_post_link( $item['order_id'], true ) );
```

Comment:
Perhaps we shouldn't `esc_url()` here and just escape closer to output (which is [already being done](https://github.com/moderntribe/event-tickets/pull/231/files#diff-cf932265bfaec1459c1d705bec1bb62cR159)).